src/test/java/org/jabref/logic/cleanup/RemoveLinksToNotExistentFilesTest.java

```diff
@@ -50,12 +54,27 @@ void setUp(@TempDir Path bibFolder) throws IOException {
         Files.createFile(bibFolder.resolve("test.bib"));
         databaseContext.setDatabasePath(bibFolder.resolve("test.bib"));
 
-        entry = new BibEntry();
-        entry.setCitationKey("Toot");
-        entry.setField(StandardField.TITLE, "test title");
-        entry.setField(StandardField.YEAR, "1989");
         LinkedFile fileField = new LinkedFile("", fileBefore.toAbsolutePath(), "");
-        entry.setField(StandardField.FILE, FileFieldWriter.getStringRepresentation(fileField));
+        
+        
+        //Entry with one online and one normal linked file
+        entry = new BibEntry(StandardEntryType.Article)
+                .withField(StandardField.AUTHOR, "Shatakshi Sharma and Bhim Singh and Sukumar Mishra")
+                .withField(StandardField.DATE, "April 2020")
+                .withField(StandardField.YEAR, "2020")
+                .withField(StandardField.DOI, "10.1109/TII.2019.2935531")
+                .withField(StandardField.FILE, FileFieldWriter.getStringRepresentation(Arrays.asList(
+                    new LinkedFile("", "https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=8801912:PDF", ""),
+                    fileField)))
```

Comment:
Is there a reason not to [just passing the string](https://github.com/JabRef/jabref/blob/main/src/test/java/org/jabref/gui/edit/ReplaceStringViewModelTest.java#L35) of adding the file?

Reply:
Because there are 2 files we want to add, one of them is the online link and other is the fileField, that's why we create a list for them first